components/MenuModal.js

```diff
@@ -1,14 +1,26 @@
-import React, { useState } from 'react'
-import styled from 'styled-components'
+import React, { useCallback, useState } from 'react'
+import styled from 'styled-components/macro'
 import Modal from 'react-bootstrap/Modal'
 import ButtonToolbar from 'react-bootstrap/ButtonToolbar'
 import Button from 'react-bootstrap/Button'
 
 function MenuModal() {
   const [show, setShow] = useState(false)
+  const [animate, setAnimate] = useState(true)
 
-  const close = () => setShow(false)
-  const toggle = () => setShow(show => !show)
+  const toggle = useCallback(() => setShow(show => !show), [])
+
+  const hide = useCallback(() => setShow(false), [])
```

Comment:
These 2 functions are related, log-wise. We can delete line 12 I think!

Reply:
Yes! Done in https://github.com/aragon/anj.aragon.org/pull/23/commits/1002e09f752f58c3d7bf1898f54a89c8a403bfa1